src/v/kafka/group_probe.h

```diff
@@ -35,17 +34,11 @@ class group_offset_probe {
           partition_label(tp.partition())};
         _metrics.add_group(
           prometheus_sanitize::metrics_name("kafka:group"),
-          {
-            sm::make_derive(
-              "offset",
-              [this] { return _offset; },
-              sm::description("Group offset"),
-              labels)
-          });
-    }
-
-    void set_offset(model::offset& offset) {
-        _offset = offset;
+          {sm::make_derive(
+            "offset",
+            [this] { return _offset; },
+            sm::description("Group offset"),
+            labels)});
```

Comment:
@ZeDRoman the commit `ducktape: add tests for group offset metric` contains the duck tape test, plus all the updates to address feedback to the c++ code. those changes should be split out and squashed into the first commit `kafka: probe to send group offset to prometheus`

Reply:
fixed

Reply:
thanks! also, you can click `resolve conversation` when you've addressed some feedback.